tests/integration/test_rpm.py

```diff
@@ -265,3 +303,18 @@ def test_e2e_rpm(
         expected_cmd_output,
         cachi2_image,
     )
+
+
+@pytest.mark.skipif(
+    os.getenv("CACHI2_TEST_LOCAL_DNF_SERVER") != "true", reason="CACHI2_TEST_LOCAL_DNF_SERVER!=true"
+)
+def test_nginx_config() -> None:
+    """
+    Ensures the local nginx server is properly configured to require client certificate authentication.
+
+    This test is important. If nginx is not properly configured rpm_with_cert_auth test might return a false positive.
```

Comment:
Is it important? We should have a SSL health check right as the container starts to make sure it is properly configured. This test isn't justified in my mind.

Reply:
It says it is important right in the docstring so it must be important, no?

This tests that the result of a request to the protected endpoint _without_ certificates returns a 400 as it should. If we don't test this and somehow the nginx certificates auth is misconfigured, you might get a false 'pass' from the test that _does_ send certificates.